internal/fs/inode/dir.go

```diff
@@ -91,6 +91,19 @@ type DirInode interface {
 		ctx context.Context,
 		tok string) (entries []fuseutil.Dirent, newTok string, err error)
 
+	// ReadEntryCores reads a batch of directory entries and returns them as a
+	// map of `inode.Core` objects along with a continuation token that can be
+	// used to pick up the read operation where it left off.
+	// Supply the empty token on the first call.
+	//
+	// At the end of the directory, the returned continuation token will be
+	// empty. Otherwise it will be non-empty. There is no guarantee about the
+	// number of entries returned; it may be zero even with a non-empty
+	// continuation token.
+	ReadEntryCores(
+		ctx context.Context,
+		tok string) (cores map[Name]*Core, newTok string, err error)
```

Comment:
Per [go/go-style/decisions#func-formatting](go/go-style/decisions#func-formatting) (externally - https://google.github.io/styleguide/go/decisions#function-formatting):
```suggestion
	ReadEntryCores(ctx context.Context, tok string) (cores map[Name]*Core, newTok string, err error)
```

Follow at other places too